### PR TITLE
Implemented StopGameMessage.

### PR DESCRIPTION
**Pull request by Ranray**

### CODE REVIEW

File: Soccar [UI]/src/nl/socnet/message/handler/StopGameMessageHandler.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+/**
+ * Handler for StopGameMessage.
+ *
+ * @author PTS34A
+ */
+public class StopGameMessageHandler extends MessageHandler<StopGameMessage> {
+
+    @Override
+    protected void handle(Connection connection, StopGameMessage message) throws Exception {
+        //TODO : Exactly stop the game.
```
Comment (TehMessiah) [inline Soccar [UI]/src/nl/socnet/message/handler/StopGameMessageHandler.java]: INFO Complete the task associated to this TODO comment. [rule](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AS1135)

**Comment by TehMessiah** [comment]
SonarQube analysis reported 1 issue
* INFO 1 info

Watch the comments in this conversation to review them.

File: Soccar [UI]/src/nl/soccar/ui/fx/controller/GameViewFXMLController.java [hunk 2]
```diff
@@ -114,7 +110,19 @@ public void spawnEntity(Entity entity) {
      * @param paused Game is paused, not null.
      */
     public void setPaused(boolean paused) {
-        engine.getGame().setPaused(paused);
+        gameCanvas.getGameEngine().getGame().setPaused(paused);
+    }
+
+    public void stop() {
```
Comment (ghost) [inline Soccar [UI]/src/nl/soccar/ui/fx/controller/GameViewFXMLController.java:116]: MINOR Document this public method. [rule](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)

**Comment by ghost** [comment]
SonarQube analysis reported 1 issue
* MINOR 1 minor

Watch the comments in this conversation to review them.

**Review by Laugslander — APPROVED** [review]

**Pull request merged**